src/features/security-cloud-game/SecurityCloudGame.js

```diff
@@ -43,7 +44,7 @@ const SecurityCloudGame = ({ height, intl: { formatMessage }, onValidDrop, width
             gridTrackSize: minGameBoardLength / GRID_TRACK_SIZE_RATIO,
         });
 
-        messageElement.focus();
+        messageElement.focus({ preventScroll: true });
```

Comment:
are there any accessibility concerns if it becomes focused but is off-screen without auto-scroll?

Reply:
I removed this line due to the UX issue mentioned above, so it should not be a concern anymore.